tests/plugins/inventory/test_netbox.py

```diff
@@ -12,37 +12,23 @@
 
 
 def get_inv(requests_mock, case, **kwargs):
-    with open("{}/{}/mocked/devices.json".format(BASE_PATH, case), "r") as f:
-        requests_mock.get(
-            "http://localhost:8080/api/dcim/devices/?limit=0",
-            json=json.load(f),
-            headers={"Content-type": "application/json"},
-        )
-    return netbox.NBInventory.deserialize(**kwargs)
+    for num in range(1, 4):
+        with open(f"{BASE_PATH}/{case}/mocked/devices_{num}.json", "r") as f:
+            requests_mock.get(
+                "http://localhost:8080/api/dcim/devices/?limit=3",
+                json=json.load(f),
+                headers={"Content-type": "application/json"},
+            )
 
+        # Something needs to be added here to combine the mocked requests
+        # within the for loop and return one set of kwargs
 
-def transform_function(host):
-    vendor_map = {"Cisco": "ios", "Juniper": "junos"}
-    host["platform"] = vendor_map[host["vendor"]]
+    return netbox.NBInventory.deserialize(**kwargs)
 
 
 class Test(object):
     def test_inventory(self, requests_mock):
-        inv = get_inv(requests_mock, "2.3.5")
```

Comment:
don't remove the old test, the old test should still pass, it's how we ensure we don't break things for existing users.